pyproject.toml

```diff
@@ -6,6 +6,9 @@ build-backend = "setuptools.build_meta"
 name = "ferny"
 version = "0"
 
+[project.optional-dependencies]
+test = ['mock-ssh-server', 'pytest']
```

Comment:
Ugh.  I have to say that I'm kinda not a fan of this.  Is there some standard that says that it ought to be done this way?

In general, I prefer to have less stuff that pip might use as an excuse to randomly download crap from the internet without asking about it...

Reply:
So you object to the declaration of test dependencies itself? (See https://peps.python.org/pep-0631/#optional-dependencies ). Or only to using pip to install them? For Fedora etc. we can use `dnf`, but for CentOS and Debian (where mock-ssh is not packaged), we'll have to call pip, I'm afraid.

Reply:
Dropped the commit, as per IRC.